integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -7094,6 +7106,17 @@ msgstr ""
 "Sie haben nicht die nötige Berechtigung, um diese Seite zu veröffentlichen, "
 "aber Sie können Änderungen vornehmen und diese zur Überprüfung vorlegen."
 
+#: cms/views/pages/page_form_view.py
+#, python-format
+msgid ""
+"The slug was changed from '%(user_slug)s' to '%(slug)s', because "
+"'%(user_slug)s' is already used by <a href='%(link)s' class='underline hover:"
+"no-underline'>%(translation)s</a>"
+msgstr ""
+"Der URL-Parameter wurde von '%(user_slug)s' zu '%(slug)s' geändert, da "
+"'%(user_slug)s' bereits von <a href='%(link)s' class='underline hover:no-"
+"underline'>%(translation)s</a> verwendet wird"
```

Comment:
```suggestion
#, python-brace-format
msgid ""
"The slug was changed from '{user_slug}' to '{slug}', because '{user_slug}' "
"is already used by <a href='{link}' class='underline hover:no-"
"underline'>{translation}</a>"
msgstr ""
"Der URL-Parameter wurde von '{user_slug}' zu '{slug}' geändert, da "
"'{user_slug}' bereits von <a href='{link}' class='underline hover:no-"
"underline'>{translation}</a> verwendet wird"
```